docs/asciidoc/faq.adoc

```diff
@@ -85,3 +86,24 @@ Format
 Reactor Netty adds information for the connection at the beginning of every log record (when this is possible).
 Use the id of the connection in order to extract all log records for a particular HTTP request.
 For more information see <<faq.logging-prefix>>
+
+[[faq.connection-closed]]
+== How can I debug "Connection prematurely closed BEFORE response"?
+By default, Reactor Netty clients use a connection pooling. When a connection is acquired from the connection pool, it is checked whether it is still opened.
+However, the connection can be closed at any time after the acquisition. There are many reasons that can cause connection to be closed.
+In most cases the setup might not be like: the client directly sends to the server, but there might be many other network components between them: proxies, load balancers etc.
+
+If on the client side you observe `Connection prematurely closed BEFORE response`, perform the checks below to identify the reason for the connection close:
+
+* Obtain https://en.wikipedia.org/wiki/Tcpdump[tcp dump] and check which peer sends `FIN`/`RST` signal
+* Check your network connection
```

Comment:
Add a period at the end. (Because each of these is a complete sentence, they require periods. One of the items having multiple sentences also forces the others to be sentences and have periods.)